doc-Installing_on_OpenShift_Container_Platform/topics/appe-saml-authentication-example.adoc

```diff
@@ -0,0 +1,103 @@
+[[configmap-settings]]
+=== External Authentication Configuration Map Settings
+
+See <<appe-saml-authentication-example>> for an example configuration map file.
+
+The configuration map includes the following parameters:
+
+* `auth-type` - The authentication type.
++
+This parameter controls which configuration files `httpd` will load upon startup. The default is `internal`. Supported values are:
++
+.`auth-type` values
+[width="100%",cols="30%,70%",options="header",]
+|====
+| Value    | External-Authentication Configuration 
```

Comment:
I didn't try to manually update these values in config map. 
When I need to enable external authentication, I do that via CloudForms UI -
 http://manageiq.org/docs/guides/external_auth.
I think this way has to be mentioned at least.

Reply:
Good idea, I will add a link to the downstream auth guide (sourced from the external_auth link above). https://access.redhat.com/documentation/en-us/red_hat_cloudforms/4.6/html/managing_authentication_for_cloudforms/external_auth Let me know if you think it needs better wording or placement :)